charts/kubeclarity/values.yaml

```diff
@@ -42,6 +42,35 @@ kubeclarity:
   service:
     type: ClusterIP
 
+  ingress:
+    enabled: false
+    labels: {}
+    annotations: {}
+
+    # Optionally use ingressClassName instead of deprecated annotation.
+    # See: https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
+    ingressClassName: ""
+
+    # As of Kubernetes 1.19, all Ingress Paths must have a pathType configured. The default value below should be sufficient in most cases.
+    # See: https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types for other possible values.
+    pathType: Prefix
+
+    hosts:
+      - host: chart-example.local
+        paths: []
+    ## Extra paths to prepend to the host configuration. This is useful when working with annotation based services.
+    extraPaths: []
```

Comment:
Looking at the template I think this is supposed to be indented along with paths, otherwise we're adding all extraPaths to every host. If that is the intention then I think we need to document here with a comment that extra paths will be prepended to *every* configured host's paths.

Reply:
I removed extraPaths as it was too generic.